scripts/newinstall.sh

```diff
@@ -31,9 +31,10 @@ LSST_MINICONDA_VERSION=${LSST_MINICONDA_VERSION:-4.5.4}
 # this git ref controls which set of conda packages are used to initialize the
 # the default conda env.
 LSST_LSSTSW_REF=${LSST_LSSTSW_REF:-fcd27eb}
+LSST_SPLENV_REF=${LSST_SPLENV_REF:-d8778d3}
```

Comment:
Why is `LSST_LSSTSW_REF` being replaced with `LSST_SPLENV_REF`?  `LSST_LSSTSW_REF` is set by jenkins as the CI system must know the current ref in use to construct the eups root path.  Also, if it is being replaced, why is `LSST_LSSTSW_REF` still declared?

Why is the default ref being changed?

Reply:
The environment is moved from lsstsw to the new git package scipipe_conda_env.
The reference is therefore changed to point to the new git package. 
The variable name also need to be changes, since the reference is not related to lsstsw git repository anymore.

The old variable LSST_LSSTSW_REF has been removed.